dangerfile.js

```diff
@@ -0,0 +1,14 @@
+const licenseAuditor = require('@brainhubeu/license-auditor');
+
+const whitelist = require('./license/whitelist');
+const blacklist = require('./license/blacklist');
+
+licenseAuditor({
+  whitelistedLicenses: whitelist,
+  blacklistedLicenses: blacklist,
```

Comment:
From what source did we get this black and whitelisted licenses

Reply:
Initially, there were empty. Then, I checked the warnings (with all the used licenses), I read the content of the licences, I added all the permissive licenses (similar to MIT) to the whitelist and all the others to the blacklist (in the blacklist there are mostly just license we don't use).

Reply:
Sound ok for now. But I recommend checking these licenses from blacklist to see if there are some we can use in OSS projects

Reply:
@RobertHebel 

IMO, the number of these licenses is so large, that it would waste our time to check them so I've checked only these which are currently used in react-carousel.